File: nodejs/aliexpressproduct.js
const sdk = require('api')('@smartproxy/v1.0#25e7913l1ow524w');

sdk.auth('SPusername', 'SPpassword');
sdk.realTimeExample({target: 'aliexpress_product', query: '1005002803336439', parse: false})
  .then(res => console.log(res))
  .catch(err => console.error(err));

File: python/aliexpressproduct.py
import requests

task_params = {
    'target': 'aliexpress_product',
    'query': '1005002803336439',
    'parse': False,
}

username = 'SPusername'
password = 'SPpassword'

response = requests.post(
    'https://scrape.smartproxy.com/v1/tasks',
    json = task_params,
    auth = (username, password)
)

print(response.text)